src/main/java/io/github/thebusybiscuit/slimefun4/core/commands/subcommands/VersionsCommand.java

```diff
@@ -36,7 +35,12 @@ public void onExecute(CommandSender sender, String[] args) {
             sender.sendMessage(ChatColors.color("&a" + Bukkit.getName() + " &2" + ReflectionUtils.getVersion()));
             sender.sendMessage("");
             sender.sendMessage(ChatColors.color("&aCS-CoreLib &2v" + SlimefunPlugin.getCSCoreLibVersion()));
-            sender.sendMessage(ChatColors.color("&aSlimefun &2v" + SlimefunPlugin.getVersion()));
+
+            String version = "&aSlimefun &2v" + SlimefunPlugin.getVersion();
+            if (SlimefunPlugin.getMetricsService().getVersion() != null)
+                version += "&7 (Metrics build: &2#" + SlimefunPlugin.getMetricsService().getVersion() + ')';
```

Comment:
With "seperate line" I meant an actual seperate line in chat. I think it should be it's own line.